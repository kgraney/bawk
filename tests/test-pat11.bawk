
/"IEND" end:uint2/ {
	print(end);
}

/"IEND" end:int2/ {
	print(end);
}
