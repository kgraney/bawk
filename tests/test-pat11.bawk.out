44610
-20926
